audio/monkeys-audio/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            fernandotcl monkeys-audio 3.99.6 release-
+revision                0
+categories              audio
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 Restrictive
+# Re license see: https://lists.debian.org/debian-legal/2007/09/msg00079.html
```

Comment:
Yeah this is a [very interesting license](https://github.com/fernandotcl/monkeys-audio/blob/master/LICENSE) and per your link [Debian doesn't like it](https://lists.debian.org/debian-legal/2007/09/msg00079.html). @jmroot What do you think of this? Do we need to mark it `NoMirror` since the license says if we don't agree to the terms we must delete the file and we cannot assume all mirror operators have read and agree to the license? Or is the license too weird to even add this software to MacPorts at all?

Reply:
@ryandesign Homebrew has it, for the record. I cannot comment on legal side.

Reply:
As is pointed out in the Debian thread, requiring deletion is beyond what can be enforced under copyright. But there's always the possibility it could be interpreted like an EULA instead, and those have been ruled enforceable in some places. Probably best to add `NoMirror` since that apparently most closely conforms to the author's wishes.

Seems like if you have a serious use for this format (and the original author is unwilling to relicense), it might almost be better to implement your own decoder.

Reply:
@jmroot @ryandesign The only software which uses this, of which I know, is `xld`, and I was thinking to implement an actual build of it (currently `xld` port installs pre-built binary).

Where should I put `NoMirror` in the portfile?

Reply:
Add it to the `license` line.

Reply:
@ryandesign Added, rebased.